src/c360_client/__init__.py

```diff
@@ -0,0 +1,36 @@
+import os
+from .dataset_cls import DatalakeClientDataset
+
+
+def get_project_config():
+    return dict(
+        tenant=os.getenv("C360_TENANT"),
+        stage=_get_stage(),
+        api_key=os.getenv("C360_API_KEY"),
+        api_url=os.getenv("C360_API_URL", _get_default_api_url()),
+        boto_endpoint_url=os.getenv("C360_BOTO_ENDPOINT_URL"),
+    )
+
+
+def _get_stage():
+    stage = os.getenv("C360_STAGE", "prod")
```

Comment:
A bit scary to default stage to `prod`

Reply:
This library absolutely need to default to `prod`. This is a client library so there's no deployment or anything to be concerned about.

Imagine if `boto3` defaults to a "staging" AWS instead of the AWS that we are working on today...